src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
```

Comment:
Let's add a default value for the health
```suggestion
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
```